tensorboard/lazy_test.py

```diff
@@ -77,6 +77,16 @@ def foo():
     foo.namedtuple
     self.assertEquals(repr(foo), "<%r via LazyModule (loaded)>" % collections)
 
+  def test_failed_load_idempotent(self):
+    expected_message = "you will never stop me"
+    @lazy.lazy_load("tensorboard.lazy_test_bad_import")
```

Comment:
the name could just be "bad" fwiw

Reply:
Good point; that would be better. (Less bad?)